src/systems/model_photo_shoot/ModelPhotoShoot.cc

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include "ModelPhotoShoot.hh"
+
+#include <random>
+#include <string>
+#include <vector>
+
+#include "ignition/gazebo/components/Joint.hh"
+#include "ignition/gazebo/components/JointAxis.hh"
+#include "ignition/gazebo/components/JointPosition.hh"
+#include "ignition/gazebo/components/JointPositionLimitsCmd.hh"
+#include "ignition/gazebo/components/JointPositionReset.hh"
+#include <ignition/common/Image.hh>
+#include <ignition/gazebo/World.hh>
+#include <ignition/gazebo/components/Name.hh>
+#include <ignition/gazebo/components/Pose.hh>
+#include <ignition/gazebo/rendering/Events.hh>
+#include <ignition/msgs.hh>
+#include <ignition/plugin/Register.hh>
+#include <ignition/rendering/Scene.hh>
+#include <ignition/gazebo/Util.hh>
+#include <ignition/rendering/Visual.hh>
+#include <ignition/rendering/WireBox.hh>
```

Comment:
I don't think this header is being used.

Reply:
Yup, I left a few unused headers from my tests :weary:.